Add reusable metric definitions to changelog (#243)
vellum-ai/vellum-client-generator: fern/docs/content/change-log/2024-04.mdx

title: Changelog | April, 2024
---

> ## Reusable Metric Definitions

Reviewer: ```suggestion
## Reusable Evaluation Metrics
```

_April 30th, 2024_

> Introducing reusable Evaluation Metrics!

Reviewer: ```suggestion
Introducing Reusable Evaluation Metrics!
```

> Evaluation Metrics can now be shared across your Test Suites making it easier for you to consistently test and evaluate your Prompt / Workflow performance.

Reviewer: ```suggestion
Evaluation Metrics can now be shared across your Test Suites making it easier for you to consistently test and evaluate your Prompt / Workflow quality.
```

> Define a suite of custom metrics tailored to your business logic and use case to save time and ensure standardized evaluations.

Reviewer: ```suggestion
Define a suite of custom metrics tailored to your business logic and use-case to save time and ensure standardized evaluation criteria.
```
